src/domain/course/CoursePageContainer.tsx

```diff
@@ -41,6 +49,29 @@ const CoursePageContainer: React.FC = () => {
 
   const course = courseData?.courseDetails;
 
+  const superEventId = getEventIdFromUrl(course?.superEvent?.internalId ?? '');
```

Comment:
shouldnt this be `getEventIdFromUrl(course?.superEvent?.internalId) ?? '';`

Reply:
no, getEventIdFromUrl cannot accept undefined